error_internal.go

```diff
@@ -0,0 +1,16 @@
+package helmclient
+
+import "strings"
+
+var (
+	noHostErrorString = "no such host"
+)
+
+// IsNoHostError asserts noHostError.
+func IsNoHostError(err error) bool {
```

Comment:
```suggestion
func isNoHostError(err error) bool {
```

It's an internal error matcher so it shouldn't be exported.